cli/util/fs.rs

```diff
@@ -687,7 +688,7 @@ impl LaxSingleProcessFsFlag {
     let open_result = std::fs::OpenOptions::new()
       .read(true)
       .write(true)
-      .create(true)
+      .truncate(true)
```

Comment:
I have a feeling this isn't correct

Reply:
> I have a feeling this isn't correct

Thank you for your reply. 
Yes you are right.
Sorry my mistake...

```sh
warning: file opened with `create`, but `truncate` behavior not defined
   --> cli\util\fs.rs:691:8
    |
691 |       .create(true)
    |        ^^^^^^^^^^^^- help: add: `.truncate(true)`
```